styles/global.module.css

```diff
@@ -0,0 +1,12 @@
+.pepoButton {
+  color: black;
+  background-color: #ffcc00;
```

Comment:
I think @longy2k had defined some theming: https://github.com/Panethnic-Pourovers/admin/blob/main/components/Theme.tsx

Reply:
Ahhh I have no idea how that works so I'll go do me some learnin